pandas/tests/series/methods/test_sort_index.py

```diff
@@ -104,18 +104,13 @@ def test_sort_index_multiindex(self, level):
         res = s.sort_index(level=level, sort_remaining=False)
         tm.assert_series_equal(s, res)
 
-    def test_sort_index_kind(self):
+    @pytest.mark.parametrize("kind", ["quicksort", "mergesort", "heapsort", "stable"])
```

Comment:
can you make this a fixture (put in the top of this test file is fine) as well.

Reply:
can you do this

Reply:
Should I just add a fixture or add it and replace the parametrize statements ?

Reply:
add a fixture in this file and use it